.github/actions/terraform-observe/opal-validation/action.yaml

```diff
@@ -0,0 +1,87 @@
+name: 'OPAL validation'
+description: 'run tf apply and destroy to validate opal'
+inputs:
+  observe-customer-id:
+    description: "customer id for test observe instance"
+    required: true
+  observe-domain:
+    description: "observeinc.com or observe-staging.com"
+    required: true
+  observe-user-handle: 
```

Comment:
Use the official nomenclature in tf. `OBSERVE_USER_EMAIL`, `OBSERVE_USER_PASSWORD`.